R/formatter.r

```diff
@@ -1,3 +1,84 @@
+#' Number formatter: a generic formatter for numbers
+#'
+#' @return `number_format` returns a function with single parameter
+#' `x`, a numeric vector, that returns a character vector
+#' @param x a numeric vector to format
```

Comment:
Can you please make parameter descriptions into sentences? i.e. start with a capital letter and end with a full stop?

Reply:
ok. I will update the PR tonight or tomorrow.

Reply:
done